docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,480 @@
+
+{% endcapture %}
+
+{% assign tab_names = "Choose one...,systemd,Static Pods" | split: ',' | compact %}
+{% assign tab_contents = site.emptyArray | push: choose | push: systemd | push: static_pods %}
+
+{% include tabs.md %}
+
+## Set up master Load Balancer
+
+1. The next step is to create a cloud Load Balancer for your master nodes. This solution will depend on your cloud provider. Examples are:
+
+    * [AWS Elastic Load Balancer](https://aws.amazon.com/elasticloadbalancing/)
+    * [GCE Load Balancing](https://cloud.google.com/compute/docs/load-balancing/)
+    * [Azure](https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview)
+
+    You will need to ensure that the load balancer routes to all three master machines set up in the previous step **on port 6443**. If possible, use a smart load balancing algorithm like "least connections", and use health checks so unhealthy nodes can be removed from circulation. Most providers will provide these features.
+
+## Acquire etcd certs
+
+Only follow this step if your etcd is hosted on dedicated nodes (**Option 1**). If you are hosting etcd on the masters (**Option 2**), you can skip this step since you've already generated the etcd certificates on the masters.
+
+1. Generate SSH keys for each of the master nodes by following the steps in the [create ssh access](#create-ssh-access) section. After doing this, each master will have an SSH key in `~/.ssh/id_rsa.pub` and an entry in `etcd0`'s `~/.ssh/authorized_keys` file.
+
+1. Next, run the following:
+
+    ```
+    mkdir -p /etc/kubernetes/pki/etcd
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca.pem /etc/kubernetes/pki/etcd
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client.pem /etc/kubernetes/pki/etcd
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client-key.pem /etc/kubernetes/pki/etcd
+    ```
+
+## Run `kubeadm init` on `master0` {#kubeadm-init-master0}
+
+1. In order for kubeadm to run, you first need to write a configuration file:
+
+    ```
+    cat >config.yaml <<EOL
+    apiVersion: kubeadm.k8s.io/v1alpha1
+    kind: MasterConfiguration
+    api:
+    advertiseAddress: <load-balancer-ip>
+    etcd:
+    endpoints:
+    - https://<etcd0-ip>:2379
+    - https://<etcd1-ip>:2379
+    - https://<etcd2-ip>:2379
+    caFile: /etc/kubernetes/pki/etcd/ca.pem
+    certFile: /etc/kubernetes/pki/etcd/client.pem
+    keyFile: /etc/kubernetes/pki/etcd/client-key.pem
+    networking:
+    podSubnet: <podCIDR>
+    apiServerCertSANs:
+    - <load-balancer-ip>
+    EOL
+    ```
+
+    Ensure that the following placeholders are replaced:
+
+    - `<load-balancer-ip>` with the public IPv4 of your load balancer. 
+    - `<etcd0-ip>`, `<etcd1-ip>` and `<etcd2-ip>` with the IP addresses of your three etcd nodes
+    - `<podCIDR>` with your Pod CIDR. Please read the [CNI network section](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network) of the docs for more information.
```

Comment:
information whether you need this or not (depends on the CNI network provider of your choice)